### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by hanzei**
#### Summary
Update dependencies for `v2.0.0` release.

#### Ticket Link
n/a

### CODE REVIEW

File: webapp/package.json [hunk 1]
```diff
@@ -13,6 +13,7 @@
     "core-js": "3.1.4",
     "debounce-promise": "3.1.2",
     "mattermost-redux": "github:mattermost/mattermost-redux#b56e7e3a6f7f5de281fd3541eefec9345a8bbc89",
+    "node-fetch": ">=2.6.1",
```
Comment (larkox) [inline webapp/package.json:16]: What is the difference between using the `^` or `>=`? Also, I thought we preferred using fixed version numbers.

**Review by larkox — APPROVED** [review]
LGTM, but I have one question.

File: webapp/package.json (hunk 1 above)
Comment (hanzei) [reply webapp/package.json]: That is a great point :+1:. Yes, we should be using fixed number. I updated `package.json` to only use fixed version and also updated all dependencies (besides `@primer/octicons-react`) while touching the file.

**Review by larkox — APPROVED** [review]

**Review by jfrerich — APPROVED** [review]
LGTM

**Comment by hanzei** [comment]
Let's test this in the release process

**Pull request merged**